### ISSUE TITLE
Fix duplicate logging

### ISSUE THREAD

**Issue by sampierson** [HumanCellAtlas/upload-service#79]
Definitely in checksum-daemon, probably elsewhere too.

**Comment by sampierson** [comment]
Isn't this fixed?

**Comment by maniarathi** [comment]
Nope... I thought I fixed it and turns out it wasn't logging everything as
expected so we reverted the PR and re-opened the ticket.
